src/api/handlers/userprofile.py

```diff
@@ -61,6 +60,7 @@ def create(self, request):
       .expect("full_name", str, is_required=True)
       .expect("preferred_name", str, is_required=True)
       .expect("is_vendor", bool, is_required=True)
+      .expect('color', str, is_required=True)
```

Comment:
Color should not be a required parameter.  This will break one other app which uses this route.